PCbuild/env.bat

```diff
@@ -10,6 +10,7 @@ rem 'v110', 'v120' or 'v140') to the build script.
 echo Build environments: x86, amd64, x86_amd64
 echo.
 set VSTOOLS=%VS140COMNTOOLS%
+if "%VSTOOLS%"=="" set VSTOOLS=%VS160COMNTOOLS%
```

Comment:
I don't think this is correct - we'll want some detection more like what's in `find_msbuild.bat` (using the `vswhere.exe` app - and if that's not there, then neither VS 2017 or VS 2019 are installed!)